Demo/Sources/Model/Template.swift

```diff
@@ -126,6 +126,48 @@ enum URLTemplate {
         title: "Unknown URL",
         type: .url(URL(string: "http://localhost:8123/unavailable/master.m3u8")!)
     )
+
+    static let bitmovinOnDemandSintel = Template(
+        title: "Sintel",
+        description: "Multiple subtitles and audio tracks",
+        type: .url(.init(string: "https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8")!)
+    )
+
+    static let bitmovinOnDemandBigBuckBunny = Template(
+        title: "Big Buck Bunny",
+        description: "4K, HEVC",
+        type: .url(.init(string: "https://cdn.bitmovin.com/content/encoding_test_dash_hls/4k/hls/4k_profile/master.m3u8")!)
+    )
+
+    static let bitmovinOnDemandAoM_1 = Template(
+        title: "Art of Motion",
+        description: "VoD, single audio track",
+        type: .url(.init(string: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8")!)
+    )
+
+    static let bitmovinOnDemandAoM_2 = Template(
```

Comment:
I guess we can drop this example. After checking again there is no FairPlay encryption on this example at all.